### DISCUSSION TITLE
Higher HRR than set with RAMP

### DISCUSSION THREAD

**Discussion by NhatNguyenEBC** [firemodels/fds#13813] [Q&A] [answered]
Hello,

The simulation involves a simple cube geometry and utilizes a time-varying heat release rate implemented with the RAMP function. However, I have observed that the output heat release rate (HRR) is consistently higher than the set HRR, and I am unable to determine the cause of this discrepancy. Based on my understanding, the specified heat release rate is translated into an injected fuel mass rate. Consequently, the combustion of this released fuel mass may lead to minor discrepancies. However, it appears to me that there exists a consistent offset between the specified HRR and the calculated HRR.

- Am I missing something in my setup? I want to reproduce experimental data where the HRR is measured. To facilitate comparison with other quantities, I would like to have matching HRRs.
- Could the problem be that my mesh is not refined enough to resolve the fire source correclty? I currently have a D*/$`\Delta x`$ value of 16 and a cell size of approximately 8.4 cm.
- Could the fluctuations in my specified HRR be the cause of this consistent discrepancy?

In black the set HRR and in red the HRR values taken from the "*_hrr.csv" file.
![Ayala_Waermeleistung](https://github.com/user-attachments/assets/9b024d09-e6f4-42b2-8b05-6f1144c02471)

My code:

[Code.txt](https://github.com/user-attachments/files/17948707/Code.txt)

Best Regards

**Comment by Er9y714** [comment]
![image](https://github.com/user-attachments/assets/1d6c767c-fcd6-43d4-852b-e76f83e6f4db)
VENT and OBSTruction of the burner is not aligned with the mesh.

**Reply by NhatNguyenEBC** [reply]
1. According to the manual, the HRR is supposed to be correct regardless of whether it's misaligned or not, right? "When the OBST snaps to the Cartesian mesh, the area of the cell face is changed, and therefore the total
area of a SURF may slightly change. If a specified mass flux boundary (including HRRPUA) is given, then
FDS will automatically adjust this mass flux so that total of the specified area times the specified mass flux
is maintained." (7.2.3) 
2. How do you access the view you see in your image?

**Reply by Er9y714** [reply]
1. I was not aware of this (in conjunction with @drjfloyd 's comment) 
2. I used Pyrosim to process the input file.

**Comment by drjfloyd** [comment, marked as answer]
@Er9y714 FDS does not require the OBST to be aligned with the grid. FDS takes the as specified area on XB and the grid snapped area to adjust the HRRPUA so the correct HRR is obtained.

@NhatNguyenEBC Double check your math for the specfied curve.  You have an HRRPUA of 2129.6, a specified area of (10.276-9.230)*(10.270-9.230)=1.07848 m2, and a peak RAMP F value of 1.530365.  2129.6 kW/m2 * 1.07848 m2 x 1.530365 = 3514.6 kW. This matches the peak FDS value in your plot and is higher than the computed value of ~3250 kW.

**Reply by Er9y714** [reply]
I did not know about this. Thank you.  I thought the snapped area is taken into account.

**Reply by NhatNguyenEBC** [reply]
You are completely right. I forgot to normalize my HRR with the surface area of the vent since it is HRRPUA. Thank you and sorry for not checking my math first.
